W-19586062 -  X12 release v2.18.0 (#3025)
mulesoft/docs-connectors: x12-edi/2.18/modules/ROOT/pages/x12-edi-connector-reference.adoc

* If `false`, the connector continues processing the received interchange numbers and leaves it to the application flow to track interchange numbers and handle them appropriately. |  true |
| Require unique groups a| Boolean a|  Whether to enforce globally unique Group Control Numbers (GS06) for received functional groups:

* If `true`, group numbers must be unique across all interchanges received from the same partner and application (as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes). 
* If `false`, group numbers must be unique within an interchange. 
|  false |
| Require unique transactions a| Boolean a|  Whether to enforce globally unique Transaction Set Control Numbers (ST02) for received transaction sets: 

* If `true`, transaction set numbers must be unique across all functional groups received from the same partner and application (as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes). 
* If `false`, transaction set control numbers must be unique within a particular functional group. |  false |

> | Require unique transactions in group a| Boolean a|  Whether to enforce globally unique Transaction Set Control Numbers (ST02) for received transaction sets with the group: 

Reviewer: ```suggestion
| Require unique transactions in group a| Boolean a|  Whether to enforce globally unique transaction set control numbers (ST02) for received transaction sets scoped within the functional group: 
```

> * If `true`, transaction set numbers must be unique within the functional groups received from the same partner and application (as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes). 

Reviewer: ```suggestion
* If true, transaction set control numbers (ST02) must be unique for each functional group. The uniqueness requirement is based on the specific partner (interchange sender and receiver IDs) and the application (functional group sender and receiver application codes) that sent it. 
```

Reviewer: @RamaKrishna-4 Is it necessary to have "(ST02)" in this bullet? It's not in the second bullet.

> * If `false`, transaction set control numbers can be duplicate within a particular functional group and they will be treated as separate transaction sets. | true |

Reviewer: ```suggestion
* If `false`, a functional group can have duplicate transaction set control numbers, and each instance is treated as a separate transaction set. | true |
```
